src/map_field.cpp

```diff
@@ -1264,6 +1267,10 @@ bool map::process_fields_in_submap( submap *const current_submap,
                         furn_set( p, f_null );
                     }
                 }
+                if( curtype == fd_insecticidal_gas ) {
```

Comment:
This is should be defined in json (and it is actually defined, but with different values):

```JSON
    "percent_spread": 100,
    "outdoor_age_speedup": "1 minutes",
    "dirty_transparency_cache": true,
```